roles/aodh/defaults/main.yml

```diff
@@ -23,14 +23,27 @@ aodh:
       - { name: PyMySQL }
       - { name: python-memcached }
       - { name: python-ceilometerclient }
-    system_dependencies: []
+    system_dependencies: 
+      ubuntu:
+        - apache2
```

Comment:
apache2 will get installed by the apache role, as you're dep'ing on it in meta/main.yml